cocotb/share/lib/embed/gpi_embed.c

```diff
@@ -219,9 +222,16 @@ int get_module_ref(const char *modname, PyObject **mod)
     return 0;
 }
 
-int embed_sim_init(gpi_sim_info_t *info)
+
+static void embed_sim_event(void*, gpi_event_t, const char*);
+static void cocotb_log_v(void* userdata, const char *name, enum gpi_log_levels level, const char *pathname, const char *funcname, long lineno, const char *msg, va_list argp);
```

Comment:
Add a line break in the argument list?

Reply:
I broke every arg out onto its own line. I'm not a fan of multiple args on each of multiple lines.